File-Formats/Presentation/Loading-and-Saving-the-Presentation.md

```diff
@@ -51,6 +51,8 @@ IPresentation pptxDoc = Presentation.Open(inputStream);
 
 {% endtabs %}
 
+You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/PowerPoint-Examples/blob/master/Read-and-save-PowerPoint-presentation/Open-and-save-PowerPoint).
```

Comment:
Change the hyperlink - https://github.com/SyncfusionExamples/PowerPoint-Examples/tree/master/Read-and-save-PowerPoint-presentation/Open-and-save-PowerPoint

Do the same for all Open-and-save-PowerPoint samples referring hyperlink.

Reply:
@Suriya-Balamurugan -Addressed